FormidableLabs/envy
# Added button to allow copying trace as cURL snippet

## What does this PR do?

- Adds a new button to the trace summary section to allow the request to be copied to the clipboard as a curl command
- Will show a brief success "toast" notification when copied
- Unit tests updated to retain 100%

## Screenshots

![image](https://github.com/FormidableLabs/envy/assets/17857833/1b6f1a50-24cf-45b0-b4f5-464a5b6a69a9)

![image](https://github.com/FormidableLabs/envy/assets/17857833/17a30e37-cf66-4ab9-8bbb-353e7d76806d)

## How to test

- Run `yarn workspace @envyjs/webui start`
- Let some traces flow through (or inject the mock traces)
- Select a trace
- Click the "Copy as cURL snippet" button
- Verify that the clipboard contains a valid cURL command
